src/app/pages/ArticlePage/ElectionBanner/index.tsx

```diff
@@ -18,22 +19,31 @@ export default function ElectionBanner({ aboutTags }: { aboutTags: Tag[] }) {
 
   if (isLite) return null;
 
-  const { iframeSrc, iframeSrcAmp, thingIds } = BANNER_CONFIG;
+  const { heights, iframeSrc, iframeDevSrc, thingIds } = BANNER_CONFIG;
 
   const validAboutTag = aboutTags?.find(tag => thingIds.includes(tag.thingId));
 
   const showBanner = validAboutTag && electionBannerEnabled;
 
   if (!showBanner) return null;
 
+  const {
+    SIMORGH_APP_ENV,
+    SIMORGH_INCLUDES_BASE_URL,
+    SIMORGH_INCLUDES_BASE_AMP_URL,
+  } = getEnvConfig();
+
+  const iframeSrcToUse = SIMORGH_APP_ENV === 'live' ? iframeSrc : iframeDevSrc;
```

Comment:
```suggestion
  const iframeSrcToUse = isLive() ? iframeSrc : iframeDevSrc;
```

Reply:
`isLive` calls `getEnvConfig` so may be a bit redundant here to use `isLive`, although more consistent with other per-environment value selection?

Reply:
Yeah that's fine. Given that this is all pretty temporary & we need to move fast 👻